microbenchmark-template/pom.xml

```diff
@@ -28,7 +28,7 @@
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>sonar-db</artifactId>
+      <artifactId>sonar-dbClient</artifactId>
```

Comment:
yes, because I used qp/search commits, it will be fixed as soon I rebase with the correction